testkit/src/main/scala/org/bitcoins/testkit/core/gen/TLVGen.scala

```diff
@@ -117,23 +144,16 @@ trait TLVGen {
   }
 
   def fundingSignaturesV0TLV: Gen[FundingSignaturesV0TLV] = {
-    for {
-      numInputs <- Gen.choose(1, 10)
-      outPoints <- Gen.listOfN(numInputs, TransactionGenerators.outPoint)
-      sigs <- fundingSignaturesV0TLV(outPoints.toVector)
-    } yield {
-      sigs
-    }
+    Gen.choose(1, 10).flatMap(fundingSignaturesV0TLV)
   }
 
-  def fundingSignaturesV0TLV(
-      outPoints: Vector[TransactionOutPoint]): Gen[FundingSignaturesV0TLV] = {
-    for {
-      sigs <- Gen.listOfN(outPoints.length,
-                          CryptoGenerators.digitalSignatureWithSigHash)
-    } yield {
-      FundingSignaturesV0TLV(outPoints.zip(sigs).toMap)
-    }
+  def fundingSignaturesV0TLV(numWitnesses: Int): Gen[FundingSignaturesV0TLV] = {
+    Gen
+      .listOfN(
+        numWitnesses,
+        WitnessGenerators.p2wpkhWitnessV0
```

Comment:
nit: Add a //TODO so we remember to change this to any kind of witness in future